Sources/armory/renderpath/RenderPathDeferred.hx

```diff
@@ -560,6 +588,7 @@ class RenderPathDeferred {
 		path.bindTarget("_main", "gbufferD");
 		path.bindTarget("gbuffer0", "gbuffer0");
 		path.bindTarget("gbuffer1", "gbuffer1");
+		path.bindTarget("gbuffer_refraction", "gbuffer_refraction");
```

Comment:
Hm, gbuffer_refraction should be bound only when it is enabled?

Reply:
fine.